### DISCUSSION TITLE
Blank screen

### DISCUSSION THREAD

**Discussion by khaliq2** [skyline-emu/skyline#2102] [Q&A] [answered]
Why when I run on the sony xperia xz3 the screen is blank, it only makes game sounds. I hope it can be fixed. 
![Screenshot_20221224-223231](https://user-images.githubusercontent.com/121345404/209442692-34f90e67-cabe-4c04-8698-f803a03392da.png)

**Comment by Crytonics** [comment, marked as answer]
Maybe that game doesn't work yet.
